src/MainWindow/main_window.cpp

```diff
@@ -453,6 +451,10 @@ void MainWindow::ReShowWindow(QString strProject) {
   updatePRViewButton(static_cast<int>(m_compiler->CompilerState()));
 
   createIpConfiguratorUI();
+
+  // Short term fix to clear any output messages at init as Compiler->Message()
```

Comment:
TODO similar to https://medium.com/babylon-engineering/todo-find-a-title-for-the-article-fee79708ca15